notebooks/tutorials/landscape_evolution/threshold_eroder/threshold_eroder.ipynb

```diff
@@ -0,0 +1,191 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "585111f0",
+   "metadata": {},
+   "source": [
+    "# User guide and example for the Landlab threshold_eroder component\n",
+    "\n",
+    "This notebook provides a brief introduction and user's guide for the threshold eroder component for landscape evolution modeling. \n",
+    "\n",
+    "Threshold eroder that cuts off slopes at a given threshold slope (Sc) and assumes material to dissolve away\n",
```

Comment:
Use $S_c$ to get inline math formatting. Suggest using left arrow instead of equals sign in equation, to indicate that this is a replacement operation rather than a statement of equality.

Reply:
fixed